src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static int count = 99;
```

Comment:
is it a constant?
constant should contain two keywords: static and final. And it should has a correct naming
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names